src/Analyzers/VisualBasic/Analyzers/RemoveUnusedParametersAndValues/VisualBasicRemoveUnusedParametersAndValuesDiagnosticAnalyzer.vb

```diff
@@ -33,6 +33,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.RemoveUnusedParametersAndValues
             Return False
         End Function
 
+        Protected Overrides Function IsClassDeclaration(node As SyntaxNode) As Boolean
+            Return False
```

Comment:
This feels weird, I believe we should check for node being a ClassDecarationSyntax here. Obviously no constructor in VB will have ClassDeclarationSyntax associated with it as VB doesn’t support primary constructors, so the modified if condition in base type will always be false for VB.